DataStructures/LinkedList/SinglyLinkedList/SinglyLinkedList.cs

```diff
@@ -153,4 +153,56 @@ public bool DeleteElement(T element)
 
         return false;
     }
+
+    /// <summary>
+    /// Deletes the first element of the list.
+    /// </summary>
+    /// <returns> true if the operation is successul.</returns>
+    public bool DeleteFirst()
+    {
+        // checks if the List is empty
+        if(Head is null)
+        {
+            return false;
+        }
+
+        // if not, the head is overwritten with the next element and the old head is deleted
+        SinglyLinkedListNode<T>? temp = Head;
+        Head = Head.Next;
+        temp = null;
+        return true;
+    }
+
+    /// <summary>
+    /// Deletes the last element of the list.
+    /// </summary>
+    /// <returns> returns true if the operation is successful. </returns>
+    public bool DeleteLast()
+    {
+        // checks if the List is empty
+        if(Head is null)
+        {
+            return false;
+        }
+
+        // checks if the List has only one element
+        else if(Head.Next is null)
```

Comment:
You don't need to have `else` keyword here because the method returns if the previous condition was `true`